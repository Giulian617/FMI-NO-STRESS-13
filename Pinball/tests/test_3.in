62 0
20 38 5
57 38 7
33 20 3
21 33 5
54 38 10
15 57 6
23 57 4
9 15 2
10 38 6
16 10 10
35 57 8
50 9 8
48 16 5
1 48 10
26 9 1
31 50 9
7 23 2
32 54 6
5 9 6
29 50 10
2 7 8
19 7 1
34 50 3
37 50 10
25 37 8
55 37 9
47 37 1
18 34 4
30 48 7
24 37 8
51 24 7
12 5 6
4 7 8
41 31 10
11 12 5
27 37 3
60 2 7
61 47 6
22 47 1
13 29 7
8 55 10
44 8 6
53 24 10
46 37 3
62 30 2
14 53 7
40 27 5
43 46 9
58 40 8
52 44 7
49 14 4
42 41 4
45 13 1
39 22 10
56 13 8
6 40 2
28 49 2
17 22 8
59 40 9
36 49 6
3 53 4
