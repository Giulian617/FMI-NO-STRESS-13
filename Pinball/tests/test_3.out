0
810635325
534363244
388081060
381313134
533053338
797664147
566795409
185606062
164720119
63552189
542152956
371212124
271043773
626458861
166316669
392992427
686770049
850986725
185472991
60310133
807893758
552324233
746364763
534363244
196220014
583333338
